src/components/UI/navbar/navbar.module.scss

```diff
@@ -4,24 +4,34 @@
   padding: 0;
   overflow: hidden;
   background-color: #1d1283;
+  display: flex;
+  align-items: center;
+  height: 5.25rem;
 }
 .infoNavbar nav li {
   float: left;
 }
 .infoNavbar nav li a {
+  font: Roboto;
+  font-size: 1.25rem;
   display: block;
-  border-radius: 0.5rem;
   color: #ffffff;
-  font-weight: bolder;
+  font-weight: 700;
   text-align: center;
-  padding: 14px 16px;
+  padding: 	0.25rem 0;
```

Comment:
Please fix the formatting here

Reply:
I was in the thought it might be my mobile issue, there is indeed a formatting issue I see

Reply:
Formatted the code. please review